Windows/GPU/WindowsGLContext.cpp

```diff
@@ -251,10 +251,15 @@ bool WindowsGLContext::Init(HINSTANCE hInst, HWND window, std::string *error_mes
 		ExitProcess(1);
 	}
 
+	if (gl_extensions.IsCoreContext)
+		glewExperimental = true;
```

Comment:
Added a comment.  See also here (but note that some extension names are omitted on Apple as well):
https://www.opengl.org/wiki/OpenGL_Loading_Library#Initialization_of_GLEW_1.13.0_and_earlier

-[Unknown]